File: Programming/perlWork/Tests/FizzBuzz/fizzbuzz.pl
#!/usr/bin/perl

my $i = 0;

LINE: while ($i lt 50)  {
   if ($i eq 0) {
      print "$i\n";
      $i++;
      next LINE;
   }
   elsif ( $i % 3 == 0 and $i % 5 == 0 )  {
      print "FizzBuzz = $i\n";
      $i++;
      next LINE;
   }
   elsif ( $i % 5 == 0 )  {
      print "Buzz = $i\n";
      $i++;
      next LINE;
   }
   elsif ( $i % 3 == 0 )  {
      print "Fizz = $i\n";
      $i++;
      next LINE;
   }
   else {
   $i++;
   print "$i\n";
   }
}

File: Programming/cppWork/Tests/FizzBuzz/fizzbuzz.cpp
#include <iostream>

#define LIMIT 100

int main()  {
   for (int i=0; i<LIMIT; i++)  {
      if ( i%5 == 0 && i%3 == 0 )  {
         std::cout << "FizzBuzz = " << i << std::endl;
         continue;
      }
      if ( i%3 == 0 )  {
         std::cout << "Fizz = " << i << std::endl;
         continue;
      }
      if ( i%5 == 0 )  {
         std::cout << "Buzz = " << i << std::endl;
         continue;
      }
   }
   return 0;
}
